packages/lesswrong/lib/collections/posts/helpers.ts

```diff
@@ -274,6 +274,8 @@ export const postCoauthorIsPending = (post: DbPost|PostsList|PostsDetails, coaut
 
 export const getConfirmedCoauthorIds = (post: DbPost|PostsList|PostsDetails): string[] => {
   let { coauthorStatuses = [], hasCoauthorPermission = true } = post;
+  if (!coauthorStatuses) return []
```

Comment:
My understanding of what's happening is that the duplicated event gets `coauthorStatuses` as `null`, and the default of `[]` only gets assigned if `coauthorStatuses` is `undefined`. So below here it tried to call `map()` on `null`.

Maybe the right thing to do here is filter out `null` values from the template event, instead of this? Not sure.